src/main/java/org/jarvis4ops/helper/JiraIssueResponseHelper.java

```diff
@@ -113,27 +114,27 @@ private void getRockstars(Map<String, Integer> jiraIssuesResponseMap, StringBuff
 		}
 	}
 	
-	public Map<String, Integer> issuesTimeElapsed(List <IssueDetails> paramIssueList)
+	public Map<String, TimeBean> issuesTimeElapsed(List <IssueDetails> paramIssueList)
 	{
-		Map <String, Integer> yesterdaysIssuesMap = new HashMap<String, Integer>();;
+		Map <String, TimeBean> yesterdaysIssuesMap = new HashMap<String, TimeBean>();;
 		
 		for (IssueDetails issue : paramIssueList)
 		{
 			String jiraDatePattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
```

Comment:
Put this pattern in a constant please - lets save heap memory (as its limited on cloud).